index.html

```diff
@@ -306,6 +339,58 @@ <h2>
     <p>
       {{DigitalCredential}} instances are [=Credential/origin bound=].
     </p>
+    <h2>
+      Extensions to `CredentialCreationOptions` dictionary
+    </h2>
+    <pre class="idl">
+    partial dictionary CredentialCreationOptions {
+      DigitalCredentialCreationOptions digital;
+    };
+    </pre>
+    <h3>
+      The `digital` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="CredentialCreationOptions">digital</dfn> member
+      allows for options to configure the issuance of a [=digital credential=].
+    </p>
+    <h2>
+      The `DigitalCredentialCreationOptions` dictionary
+    </h2>
+    <p>
+      The {{DigitalCredentialRequest}} dictionary represents a [=digital
+      credential/issuance request=]. It is used to specify an [=digital
+      credential/exchange protocol=] and a [=digital credential/credential offer=],
+      which the user agent MAY forward to a software used by a holder, such as
+      a digital wallet.
+    </p>
+    <pre class="idl">
+    dictionary DigitalCredentialCreationOptions {
+      required DOMString protocol;
+      required object data;
+    };
+    </pre>
+    <h3>
+      The `protocol` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="DigitalCredentialCreationOptions">protocol</dfn> member
+      denotes the [=digital credential/exchange protocol=] when issuing a
+      digital credential.
+    </p>
+    <p>
+      The {{DigitalCredentialCreationOptions/protocol}} member's value is be one of the
+      well-defined keys defined in [[[#protocol-registry]]] or any other custom
+      one.
+    </p>
+    <h3>
+      The `data` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="DigitalCredentialCreationOptions">data</dfn> member is the
+      [=digital credential/credential offer=] to be handled by the holder's
+      software, such as a digital wallet.
```

Comment:
Including @marcoscaceres' [suggestion](https://github.com/WICG/digital-credentials/pull/204/files#r1976434655)
```suggestion
      The <dfn data-dfn-for="DigitalCredentialCreationOptions">data</dfn> member is a
      [=digital credential/credential offer=] that conforms to some standard in the
      [[[#protocol-registry]]], to be handled by [=holder=]
      software, such as a digital wallet.
```